hack/ci-utils/go_generate.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set -o errexit
+
+
+main() {
+
+    # Variables checked for timeout values, defaults used if unset
+    soft_timeout="${GO_GENERATE_SOFT_TIMEOUT:-20m}"
+    hard_timeout="${GO_GENERATE_HARD_TIMEOUT:-21m}"
+
+    if [[ -n $CI ]]; then
```

Comment:
You can probably leave out the `if` statement since I think it's assumed that this script is specifically for CI (hack/ci-utils/*) and we'd always expect this to be set.